### PR TITLE
Release notes 2016-08-22

### PR DESCRIPTION
**Pull request by orangejulius**
Easy one this time!

### CODE REVIEW

**Comment by dianashk** [comment]
LGTM

File: release-notes.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+## 22 August 2016
+
+No changes in functionality at all, just the freshest data! We did clean up some tests and do other work only visible to developers and those who run their own Pelias instance, but nothing major.
+
+Stay tuned for next weeks release where we already have some nice changes queued up.
```
Comment (trescube) [inline release-notes.md]: `weeks` -> `week's`

**Comment by trescube** [comment]
Otherwise, :shipit:

File: release-notes.md (hunk 1 above)
Comment (orangejulius) [inline release-notes.md]: Fixed!

**Pull request merged**